src/pybind/mgr/dashboard/frontend/cypress.json

```diff
@@ -18,5 +18,9 @@
       "mochaFile": "cypress/reports/results-[hash].xml"
     }
   },
-  "retries": 1
+  "retries": 1,
+  "env": {
+    "LOGIN_USER": "admin",
+    "LOGIN_PASSWORD": "admin"
```

Comment:
If this PR is indeed the cause, my thought is that "LOGIN_PASSWORD" should be "LOGIN_PWD" to match what's written below.

Reply:
@ljflores Yup, that's correct looks like it somehow missed our radar!